### PR TITLE
add ModelValidator

### PR DESCRIPTION
**Pull request by yuliiahonchar**
change java version to 17

### CODE REVIEW

File: src/main/java/org/realworld/utils/ModelValidator.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+import jakarta.validation.Validation;
+import jakarta.validation.ValidatorFactory;
+import org.hibernate.validator.messageinterpolation.ParameterMessageInterpolator;
+
+import java.util.Set;
+
+public class ModelValidator {
+
+
+    public <T> Set<ConstraintViolation<T>> isValid(T toValidate) {
```
Comment (vhonchar) [inline src/main/java/org/realworld/utils/ModelValidator.java]: should it be `validate`?

File: src/main/java/org/realworld/utils/ModelValidator.java [hunk 2]
```diff
@@ -0,0 +1,20 @@
+package org.realworld.utils;
+
+import jakarta.validation.ConstraintViolation;
+import jakarta.validation.Validation;
+import jakarta.validation.ValidatorFactory;
+import org.hibernate.validator.messageinterpolation.ParameterMessageInterpolator;
+
+import java.util.Set;
+
+public class ModelValidator {
```
Comment (vhonchar) [inline src/main/java/org/realworld/utils/ModelValidator.java]: if this class is in package utils, better to name it with suffix "Utils".  
Also, methods usually are static in utils classes

File: src/main/java/org/realworld/utils/ModelValidator.java (hunk 1 above)
Comment (yuliiahonchar) [reply src/main/java/org/realworld/utils/ModelValidator.java]: done

File: src/main/java/org/realworld/utils/ModelValidator.java (hunk 2 above)
Comment (yuliiahonchar) [reply src/main/java/org/realworld/utils/ModelValidator.java]: done

**Review by vhonchar — APPROVED** [review]

**Pull request merged**